pkgs/data/misc/adapta-backgrounds/default.nix

```diff
@@ -11,7 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "04hmbmzf97rsii8gpwy3wkljy5xhxmlsl34d63s6hfy05knclydj";
   };
 
-  nativeBuildInputs = [ meson ninja pkg-config glib ];
+  strictDeps = true;
+  nativeBuildInputs = [ meson ninja pkg-config ];
+  buildInputs = [ glib ];
```

Comment:
Can you comment on why `glib` needs to be in `buildInputs`?

Reply:
glib is a runtime dependency, as suggested by meson.
```
adapta-backgrounds-riscv64-unknown-linux-gnu> Run-time dependency glib-2.0 found: NO (tried pkgconfig)
adapta-backgrounds-riscv64-unknown-linux-gnu> meson.build:11:0: ERROR: Dependency "glib-2.0" not found, tried pkgconfig
```